examples/cap-extSL.jl

```diff
@@ -0,0 +1,26 @@
+# ~/~ begin <<docs/src/ca-with-production.md#examples/cap-extSL.jl>>[init]
+using CarboKitten.CaProd
+using DataFrames
+using CSV
+using Interpolations
+#datasl = CSV.read("data/bs92-sealevel-curve.csv", DataFrame, header=1)
```

Comment:
We use version control to keep track of changes, please don't use comments to hide old code.

Reply:
thx!

Reply:
You still have comments in the current version